sdk/pinocchio/src/account_info.rs

```diff
@@ -1,5 +1,4 @@
 //! Data structures to represent account information.
-
 use core::{marker::PhantomData, mem::ManuallyDrop, ptr::NonNull, slice::from_raw_parts_mut};
 
 use crate::{program_error::ProgramError, pubkey::Pubkey, syscalls::sol_memset_};
```

Comment:
I am surprised clippy did not complain, but was expecting to have to put the `syscalls::sol_memset_` under a `cfg` flag:
```rust
#[cfg(target_os = "solana")]
use crate::syscalls::sol_memset_;
```